velox/exec/HashProbe.cpp

```diff
@@ -128,13 +128,7 @@ HashProbe::HashProbe(
 
   auto numKeys = joinNode_->leftKeys().size();
   keyChannels_.reserve(numKeys);
-  hashers_.reserve(numKeys);
-  for (auto& key : joinNode_->leftKeys()) {
-    auto channel = exprToChannel(key.get(), probeType_);
-    keyChannels_.emplace_back(channel);
```

Comment:
`keyChannels_` is not populated anymore

Reply:
That's a good point. Let me fix. Thanks.